tests/cli/test_integrations.py

```diff
@@ -14,37 +18,57 @@ def test_integrations_uninstall(cli_runner):
     register_groups(cli)
     result = cli_runner.invoke(
         cli,
-        ["integrations", "uninstall", "--no-aws-permissions-check"],
+        [
+            "integrations",
+            "uninstall",
+            "--no-aws-permissions-check",
+            "--nr-account-id",
+            "12345678",
+        ],
         env={"AWS_DEFAULT_REGION": "us-east-1"},
-        input="y",
+        input="y\ny",
```

Comment:
do you mean `y/n`?

Reply:
No. The `y\ny` is inputting a `y` at each of the two confirmation prompts. The `\n` is the carriage return for the first prompt. The input gets written to stdin and click (the CLI library) parses it when it presents the confirm prompts.